schemas/Quote.json

```diff
@@ -28,7 +28,21 @@
       "description": "Time in seconds between proposed_at and expires_at. Set in quotes from payment systems but not valid in actual transfers",
       "$ref": "NonNegativeDuration.json"
     },
-    "destination_expiry_duration": { "$ref": "NonNegativeDuration.json" }
+    "destination_expiry_duration": { "$ref": "NonNegativeDuration.json" },
+    "additional_info": {
```

Comment:
If you have a property called `additional_info` it should be schemaless. The point of having an `additional_info` is that it's not part of the schema and you can easily add stuff to it, but you should also not rely on it existing.